includes/class-wc-taxjar-integration.php

```diff
@@ -329,7 +375,7 @@ public function get_settings( $current_section = '' ) {
 						'type' => 'title',
 					);
 					$settings[] = array(
-						'title'    => __( 'Ship From Address', 'wc-taxjar' ),
+						'title'	=> __( 'Ship From Address', 'wc-taxjar' ),
```

Comment:
@dallendalton Was this intended?

Reply:
No it was not, it is resolved in the latest commit.